website/docs/foundations/typography/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,195 @@
+## How typography is referenced
+
+Typography styles change based on the tool. This overview explains how HDS typography standards work in each tool.
+
+In Figma, reusable text styles are called “Text Styles” and are categorized and stored in the “text panel” to be used in text-based elements. Text styles directly reference font family, weight, line height, size, and many other text-based properties within a single style and are manually maintained. Some Figma styles include:
+
+- `Display/500/Bold`
+- `Body/300/Regular`
+- `Code/300/Regular`
+
+In code, these styles are constructed differently using “Tokens.” Tokens are generated from an agnostic JSON file and differ from Figma “Text Styles” because a single token does not reference multiple font properties. Instead, each Token defines the different properties of a single text style. This means that constructing the equivalence of Figma’s Text Style `Display/500/Bold` would use the following tokens:
+
+- `font-family: var(--token-typography-display-500-font-family);`
+- `font-size:var(--token-typography-display-500-font-size);`
+- `line-height:var(--token-typography-display-500-line-height);`
+- `font-weight:var(--token-typography-font-weight-bold);`
+
+Rather than calling upon each of these tokens in code every time text is styled, the [Text Component](/components/text) was created to replicate how Figma styles group text-based properties. Additional [CSS helper classes](https://helios.hashicorp.design/foundations/typography?tab=code#css-helper-classes) have been created to ease the style creation process, if necessary.
+
+## Font families
+
+In code, Helios uses default system fonts because they are stable, support internationalization, and improve performance by reducing resource load—key for large applications at HashiCorp.
+
+In Figma, our design tooling and libraries use SF Pro font family exclusively and we recommend using this font stack for consistency.
+
+While text styling varies slightly across operating systems, users will have a consistent and expected experience within each OS.
+
+### Sans-serif
+
+#### Mac + iOS sans-serif font
+
+Large text (20px and above): SF Pro Display
+
+![Sans-serif display on Mac](/assets/foundations/typography/typography-sans-serif-display-mac.png)
```

Comment:
I should have said this in my previous round of comments, but now that the informational content is moved out of the images I think we can make all the font sample images decorative.

```suggestion
![](/assets/foundations/typography/typography-sans-serif-display-mac.png)
```